community/kernel/src/main/java/org/neo4j/kernel/builtinprocs/SchemaProcedure.java

```diff
@@ -79,8 +81,9 @@ public GraphResult buildSchemaGraph()
                 ArrayList<String> indexes = new ArrayList<>();
                 while ( indexDescriptorIterator.hasNext() )
                 {
-                    indexes.add( statementTokenNameLookup
-                            .propertyKeyGetName( indexDescriptorIterator.next().getPropertyKeyId() ) );
+                    IndexDescriptor index = indexDescriptorIterator.next();
+                    String[] propertyNames = PropertyNameUtils.getPropertyKeys( statementTokenNameLookup, index.descriptor() );
+                    indexes.add( Arrays.stream( propertyNames ).collect( Collectors.joining( "," ) ) );
```

Comment:
Arrays.toString adds extra `[]` characters, but we will try `String.join()` instead which should be better.